tests.py

```diff
@@ -17,7 +18,22 @@
     from io import StringIO
 
 
-class CompatibilityMixin:
+def skipIf(condition, reason):
+    def wrapper(fn):
+        if condition:
+            def empty_test(case):
+                pass
```

Comment:
Thinking out loud: maybe rename `case` to `self` for clarity, then call `self.skipTest(reason)` if it exists, so it gets reported?

Or maybe let's just try using the existing [unittest.skipIf](https://docs.python.org/2/library/unittest.html#unittest.skipIf), if it exists?

```python
try:
    from unittest import skipIf
except ImportError:
    def skipIf(...)
        ...
```